shared/dataflow/codeql/dataflow/internal/DataFlowImpl.qll

```diff
@@ -3945,12 +3932,6 @@ module MakeImpl<InputSig Lang> {
       override predicate isSource() { none() }
 
       override string toString() { result = sourceGroup }
-
-      override predicate hasLocationInfo(
```

Comment:
These classes ought to have the empty location instead, right? Removing this means that they now have no location, which might be an undesirable change. I guess source- and sink-groups aren't really used that much, but still.

Reply:
You can just add
```
override Location getLocation() { result.hasLocationInfo("", 0, 0, 0, 0) }
```

Reply:
I can do that, but there's no guarantee that such a location exists.

Reply:
True, but it's a lot better than guaranteeing that there's no location.